README.md

```diff
@@ -89,54 +92,94 @@ Create a truth table for the expression A || B.
-| true  | false | false  |      |
-| false | false | false  |      |
+| true  | true  | true   |   true     |
+| false | true  | false  |   true     |
+| true  | false | false  |   true     |
+| false | false | false  |   false    |
 
 ## Problem Four
 
 Create a truth table for the expression !A && !B.
 
 For reference, here is a truth table for the expression A && !B. You can fill out the last column. Don't worry about keeping the spacing exact.
 
-| A     | B     | !B    | A && !B | !A && !B |
-| ----- | ----- | ----- | ------- | -------- |
-| true  | true  | false | false   |          |
-| false | true  | false | false   |          |
-| true  | false | true  | true    |          |
-| false | false | true  | false   |          |
+| A     | B     | !B    |  !A    | A && !B | !A && !B |
+| ----- | ----- | ----- |------  | ------- | -------- |
+| true  | true  | false |  false |  false  |  false   |
+| false | true  | false |  true  |  false  |  false   |
+| true  | false | true  |  flase |  true   |  false   |
+| false | false | true  |  true  |  false  |  true    |
 
 ## Problem Five
 
 Create a truth table for the expression !(A || B).
 
+| A     | B     |  A ll B    | !(A ll B)     |
+| ----- | ----- | --------   | ----------    |
+| true  | true  |  true      |  false        |
+| false | true  |  true      |  false        |
+| true  | false |  true      |  false        |
+| false | false |  false     |  true         |
+        
+
 ## Problem Six
 
 Write a step-by-step evaluation for the following expression ([remember order of operations](https://www.mathsisfun.com/operation-order-pemdas.html)): `2 + 3 * 2 + 1`.
 For reference, here is a exp of a step-by-step evaluation:
 
 ```js
 1 + 2 + 3 + 4
+//step 1: 1 + 2 =3
 3 + 3 + 4
+//step 2: 3 + 3 = 6
 6 + 4
+//step 3: 6 + 4 = 10
 10
+//The returned value is 10
 ```
-
+/*
+7 + (6 * 52 + 3)
+step 1: 6*52 = 312
+step 2: 312 + 3 = 315
+step 3: 315 + 7 =322
+The returned value is 322
+*/
 ## Problem Seven
 
 Write a step-by-step evaluation for the following expression (remember order of operations): `4 / 2 + 8 / 4`.
-
+/*
+step 1: 4 / 2 = 2
+The returned value is: 2 + 8 / 4
+step 2: 8 / 4 = 2
+The returned value is: 2 + 2
+step 3: 2 + 4 = 4 
+The final returned value is: 4
+*/
 ## Problem Eight
 
 Write a step-by-step evaluation for the following expression: `'ca' + 'ter' + 'pi' + 'llar'`.
-
+/*
+step 1: 'ca' + 'ter' = 'cater'
+step 2: 'pi' + 'llar' = 'pillar'
+step 3: 'cater' + 'pillar' = 'caterpillar'
+*/
 ## Problem Nine
 
 Write a step-by-step evaluation for the following expression: `2 * 4 === 8 && 'car' + 'pool' === 'carpool'`.
-
+/*
+step 1:  2 * 4 = 8
+step 2: 8 === 8 => true
+step 3: 'car' + 'pool' = 'carpool'
+step 4: 'carpool' === 'carpool' => true
+step 5: true && true => true
+*/
 ## Problem Ten
 
 Write a step-by-step evaluation for the following expression: `'1' + '2' + '3' - '1'`.
+/*
+step 1: '1' + '2' = '12'
+step 2: '12' + '3' = '123'
+step 3: '123' - '1' = '122'
```

Comment:
Close! The last line here with the minus sign does a type conversion from string to number, so the end result is 122 being a number.

Reply:
You are right sir.
step 3: 123 -1 = 122 
the end result is a number